Patches/Vanilla Animals Expanded - Vehicles/ThingDefs_Races/Races_AFVehicles.xml

```diff
@@ -0,0 +1,201 @@
+						<power>24</power>
+						<cooldownTime>3.17</cooldownTime>
+						<linkedBodyPartsGroup>VAEAF_FrontGrilleGroup</linkedBodyPartsGroup>
+						<armorPenetrationBlunt>10</armorPenetrationBlunt>
+					  </li>
+					</tools>
+				  </value>
+				</li>
+				
+				<!--Delivery Truck-->
+				
+				<li Class="PatchOperationAdd">
+				  <xpath>/Defs/ThingDef[defName="VAEAF_CarDelivery"]/statBases</xpath>
+				  <value>
+				    <MeleeDodgeChance>0.10</MeleeDodgeChance>
+					<MeleeCritChance>0.20</MeleeCritChance>
+					<MeleeParryChance>0.3</MeleeParryChance>
+					<SmokeSensitivity>0.05</SmokeSensitivity>
+					<ArmorRating_Sharp>2</ArmorRating_Sharp>
+					<ArmorRating_Blunt>3</ArmorRating_Blunt>
+				  </value>
+				</li>
+				
+				<li Class="PatchOperationReplace">
+				  <xpath>/Defs/ThingDef[defName="VAEAF_CarDelivery"]/tools</xpath>
+				  <value>
+				    <tools>
+					  <li Class="CombatExtended.ToolCE">
+						<label>head</label>
+						<capacities>
+						  <li>Blunt</li>
+						</capacities>
+						<power>34</power>
+						<cooldownTime>5.96</cooldownTime>
+						<linkedBodyPartsGroup>VAEAF_FrontGrilleGroup</linkedBodyPartsGroup>
+						<armorPenetrationBlunt>14.7</armorPenetrationBlunt>
+					  </li>
+					</tools>
+				  </value>
+				</li>
+				
+				<!--Vintage Car-->
+				
+				<li Class="PatchOperationAdd">
+				  <xpath>/Defs/ThingDef[defName="VAEAF_CarVintage"]/statBases</xpath>
+				  <value>
+				    <MeleeDodgeChance>0.15</MeleeDodgeChance>
+					<MeleeCritChance>0.25</MeleeCritChance>
+					<MeleeParryChance>0.35</MeleeParryChance>
+					<SmokeSensitivity>0.05</SmokeSensitivity>
+					<ArmorRating_Sharp>4</ArmorRating_Sharp>
+					<ArmorRating_Blunt>6</ArmorRating_Blunt>
+				  </value>
+				</li>
+				
+				<li Class="PatchOperationReplace">
+				  <xpath>/Defs/ThingDef[defName="VAEAF_CarVintage"]/tools</xpath>
+				  <value>
+				    <tools>
+					  <li Class="CombatExtended.ToolCE">
+						<label>head</label>
+						<capacities>
+						  <li>Blunt</li>
+						</capacities>
+						<power>36</power>
+						<cooldownTime>2.54</cooldownTime>
+						<linkedBodyPartsGroup>VAEAF_FrontGrilleGroup</linkedBodyPartsGroup>
+						<armorPenetrationBlunt>15.625</armorPenetrationBlunt>
+					  </li>
+					</tools>
+				  </value>
+				</li>
+				
+				<!--Pickup Truck-->
+				
+				<li Class="PatchOperationAdd">
+				  <xpath>/Defs/ThingDef[defName="VAEAF_CarPickup"]/statBases</xpath>
+				  <value>
+				    <MeleeDodgeChance>0.15</MeleeDodgeChance>
+					<MeleeCritChance>0.20</MeleeCritChance>
+					<MeleeParryChance>0.40</MeleeParryChance>
+					<SmokeSensitivity>0.05</SmokeSensitivity>
+					<ArmorRating_Sharp>4</ArmorRating_Sharp>
+					<ArmorRating_Blunt>6</ArmorRating_Blunt>
+				  </value>
+				</li>
+				
+				<li Class="PatchOperationReplace">
+				  <xpath>/Defs/ThingDef[defName="VAEAF_CarPickup"]/tools</xpath>
+				  <value>
+				    <tools>
+					  <li Class="CombatExtended.ToolCE">
+						<label>head</label>
```

Comment:
This should probably be swapped to something else, given that the vehicles don't have "heads". Something like grille? body?

Reply:
Ahh, nice catch. Should be fixed now.